src/certificateForm/FormCertificate.js

```diff
@@ -0,0 +1,60 @@
+import React, { useState } from 'react'
+import { getISODate } from '../utils'
+
+export default function FormCertificate() {
+  const today = getISODate(new Date())
+  const [certificate, setCertificate] = useState({})
+
+  function handleTitleOnChange(event) {
```

Comment:
Is there maybe a possibility to write an higher order function? In that case you would not need to have 3x the same function with just one word changing.

Reply:
Yes, I want to create a Component for that